ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ConfigDetails.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+    Alert,
+    Bullseye,
+    Card,
+    CardBody,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Flex,
+    Spinner,
+} from '@patternfly/react-core';
+
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import NotifierConfigurationView from 'Components/NotifierConfiguration/NotifierConfigurationView';
+import {
+    getBodyDefault,
+    getSubjectDefault,
+    getTimeWithHourMinuteFromISO8601,
+} from '../compliance.scanConfigs.utils';
+import ScanConfigParametersView from './ScanConfigParametersView';
+import ScanConfigClustersTable from './ScanConfigClustersTable';
+import ScanConfigProfilesView from './ScanConfigProfilesView';
+
+const headingLevel = 'h2';
+
+function ConfigDetails({ isLoading, error, scanConfig }) {
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const isComplianceReportingEnabled = isFeatureFlagEnabled('ROX_COMPLIANCE_REPORTING');
+
+    if (isLoading) {
+        return (
+            <Bullseye>
+                <Spinner />
+            </Bullseye>
+        );
+    }
+
+    if (error) {
+        return (
+            <Alert variant="warning" title="Unable to fetch scan schedule" component="p" isInline>
+                {getAxiosErrorMessage(error)}
+            </Alert>
+        );
+    }
+
+    if (!isLoading && scanConfig) {
```

Comment:
Is `!isLoading` redundant here because `if (isLoading) return …` above?

Reply:
Ah, I copy-pasted the same logic. We can fix that too.